clang/lib/Driver/ToolChains/Gnu.cpp

```diff
@@ -603,10 +603,17 @@ void tools::gnutools::Linker::ConstructJob(Compilation &C, const JobAction &JA,
     // Go through the Inputs to the link.  When a listfile is encountered, we
     // know it is an unbundled generated list.
     for (const auto &II : Inputs) {
-      if (II.getType() == types::TY_Tempfilelist) {
+      // TODO: Incoming file from the unbundling of the AOCX archive is
```

Comment:
I do not understand this TODO comment

Reply:
Ideally the incoming file type should be represented as a list, and not an object.  This needs to be improved upon, but it requires more manipulation of the file as the aocx file is passed around.  I can clarify the comment.